tests/integration/test_kube_ovn.py

```diff
@@ -407,6 +408,86 @@ async def test_multi_nic_ipam(multi_nic_ipam):
     assert ip_address(iface_addrs["net1"][0]["local"]) in ip_network("10.123.123.0/24")
 
 
+class TCPDumpError(Exception):
+    pass
+
+
+@retry(
+    retry=retry_if_exception_type(TCPDumpError),
+    stop=stop_after_delay(60 * 10),
+    wait=wait_fixed(1),
+    before=before_log(log, logging.INFO),
+)
+async def run_tcpdump_test(ops_test, unit, interface, capture_comparator):
+    juju_cmd = f"ssh {unit.name} -- sudo timeout 5 tcpdump -ni {interface}"
+    retcode, stdout, stderr = await ops_test.juju(
+        *shlex.split(juju_cmd),
+        check=False,
+    )
+
+    # In GH actions, the output is in stderr and stdout is empty
+    output = stdout + stderr
+    # Timeout return code is 124 when command times out
+    if retcode == 124:
+        # Last 3 lines of stdout look like this:
+        # 0 packets captured
+        # 0 packets received by filter
+        # 0 packets dropped by kernel
+        for line in output.split("\n"):
+            if "packets captured" in line:
+                captured = int(line.split(" ")[0])
```

Comment:
fortunately tcpdump doesn't get cute and do something like `1,000 packets captured`

example)
```bash
9003 packets captured
9010 packets received by filter
7 packets dropped by kernel
```

you should be good here..